libs/vkd3d/command.c

```diff
@@ -3417,6 +3568,9 @@ static bool d3d12_command_list_begin_render_pass(struct d3d12_command_list *list
 
     list->current_render_pass = vk_render_pass;
 
+    /* Emit deferred clears with vkCmdClearAttachment */
+    d3d12_command_list_emit_render_pass_clears(list, false);
```

Comment:
It is technically possible to use LOAD_OP_CLEAR here and rely on compatible render pass rules with PSOs?

Reply:
Not without a major rework of how we obtain the render pass from the pipeline.

Reply:
Ok, good to know.